Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,94 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+from mentii import user_ctrl
+
+class MentiiAuthentication:
```

Comment:
Why is this a class if everything is marked with @staticmethod? You can import it as a module and just have top level functions. Was there naming conflicts?